packages/truffle-contract/lib/subscriptions.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * This code is temporarily unused pending resolution of bugs at Web3 & Geth that make
+ * subscriptions for the block head unreliable.
+ */
+
```

Comment:
Let's remove it, then!

You can hold me accountable to find it for you in commit history later.